src/App.js

```diff
@@ -4,7 +4,7 @@ import { Container, Card, Button, Row, Col } from 'react-bootstrap';
 import RocketLaunchDetails from './components/RocketLaunchDetails';
 import querystring from 'querystring';
 import './App.css';
-import load from './loadRocket.gif';
+import load from './transparent-rocket.gif';
```

Comment:
change the file name to the previous one i.e `loadRocket.gif` otherwise, it will raise conflicts for other PRs. We should keep file names like `loader.gif` a generic name. For now `loadRocket.gif` is fine. Change the width and height of the loader so that it's only taking space around the rocket with a transparent background. So, it works for all screen sizes and not take that extra space. let me know if you have any doubt or question

Reply:
ok having a look now

Reply:
Done- new commits on this branch.